.node-version

```diff
@@ -1 +1 @@
-14
+14.15
```

Comment:
Maybe that's [just not possible with nodenv](https://github.com/nodenv/nodenv/issues/99#issuecomment-416222725). In this case it might be better to define a full version here.

Reply:
thanks for the link. I will need to spend some time on it, later :)